apps/webapp/src/state/send.ts

```diff
@@ -84,9 +105,11 @@ export const createSendSlice = (): SliceCreator<SendSlice> => (set, get) => {
   };
 };
 
-const planWitnessBuildBroadcast = async ({ amount, recipient, selection, memo }: SendSlice) => {
-  if (typeof selection?.accountIndex === 'undefined') throw new Error('no selected account');
-  if (!selection.asset) throw new Error('no selected asset');
+type TransactionProps = Pick<SendSlice, 'amount' | 'recipient' | 'selection' | 'memo'>;
+
+const getPlan = async ({ amount, recipient, selection, memo }: TransactionProps) => {
```

Comment:
Extracted a `getPlan` function, which we call to refresh the fee whenever a form input changes.